### DISCUSSION TITLE
v0.11.2

### DISCUSSION THREAD

**Discussion by alexarchambault** [almond-sh/almond#803] [General]
## Fixes

- Keep an empty `matches` field in completion requests (#785, thanks to @kiendang)
- Fix spurious errors while defining classes or object, when the variable inspector is enabled (#793, thanks to @kiendang)

<hr /><em>This discussion was created from the release <a href='https://github.com/almond-sh/almond/releases/tag/v0.11.2'>v0.11.2</a>.</em>

**Comment by kiendang** [comment]
I notice the website hasn't been updated for the last few releases. It's still show 0.10.9. Caching issue maybe?
